lib/connection.js

```diff
@@ -548,11 +549,13 @@ Connection.prototype.handleRow = function (header, row, meta, rowLength, flags)
     clearTimeout(handler.timeout);
     handler.timeout = null;
   }
-  handler.rowIndex = handler.rowIndex || 0;
-  var rowCallback = handler.options && handler.options.rowCallback;
-  if (rowCallback) {
-    rowCallback(handler.rowIndex++, row, rowLength);
+  if (!handler.rowCallback) {
+    return this.invokeCallback(
+      handler,
+      new errors.DriverInternalError('RowCallback not found for streaming frame handler'));
```

Comment:
Is this technically even possible?  I've tried using eachRow without a row callback and the logic in parseCommonArgs is exercised.   Still good to have this here i suppose :)